src/modules/audio/openal/Audio.cpp

```diff
@@ -293,6 +305,17 @@ std::vector<love::audio::Source*> Audio::pause()
 	return Source::pause(pool);
 }
 
+void Audio::pauseContext()
+{
+	alcMakeContextCurrent(nullptr);
```

Comment:
Does anything need to be done with the audio pool thread? I'm not sure whether it will do the right thing after this happens (eg will any audio be lost from streaming sources?)

Reply:
My tests so far on iOS 12 and 14 don't show any issues. Audio resumes properly when the context is restored. 
I'm going to test some more with streaming loopable sources for interruptions longer than the sound duration.

Reply:
I've tested it with streaming loopable sources and it works fine.